docs/Community-Powered By.md

```diff
@@ -0,0 +1,21 @@
+<!-- TOC -->
+
+- [Powered By](#powered-by)
+    - [Project and Product names using "IoTDB"](#project-and-product-names-using-iotdb)
+    - [Companies and Organizations](#companies-and-organizations)
+
+<!-- /TOC -->
+## Powered By
+
+### Project and Product names using "IoTDB"
+
+Organizations creating products and projects for use with Apache IoTDB, along with associated marketing materials, should take care to respect the trademark in “Apache IoTDB” and its logo. Please refer to [ASF Trademarks Guidance](https://www.apache.org/foundation/marks/) and associated [FAQ](https://www.apache.org/foundation/marks/faq/) for comprehensive and authoritative guidance on proper usage of ASF trademarks.  
+Names that do not include “IoTDB” at all have no potential trademark issue with the IoTDB project. This is recommended.  
+Names like “IoTDB BigDataProduct” are not OK, as are names including “IoTDB” in general. The above links, however, describe some exceptions, like for names such as “BigDataProduct, powered by Apache IoTDB” or “BigDataProduct for Apache IoTDB”.  
+It is common practice to create software identifiers (Maven coordinates, module names, etc.) like “iotdb-tool”. These are permitted. Nominative use of trademarks in descriptions is also always allowed, as in “BigDataProduct is a subproduct for Apache IoTDB”.
+
+### Companies and Organizations
+To add yourself to the list, please email dev@iotdb.apache.org with your organization name, URL, a list of which IoTDB components you are using, and a short description of your use case.  
```

Comment:
```suggestion
To add yourself to the list, please email dev@iotdb.apache.org with your organization name, URL, a list of IoTDB components you are using, and a short description of your use case.  
```
As an Apache project, I would recommend lowering the bar of joining the dev@. It is a great opportunity to grow the community. They could simply subscribe to the list without any additional requirement.